app/views/posts/all.html.erb

```diff
@@ -5,7 +5,11 @@
 
  <%= link_to user_post_path(@user, post) do %>
  <p>
-  <%= "#{post.message} - #{post.created_at.to_date}" %>
+ <% if "#{post.message}".include?("\n") %>
+ <%= simple_format("#{post.message} - #{post.created_at.to_date}") %>
```

Comment:
I don't think I am understanding the simple_format method.
Can you not use that formatting for each post and thus avoid the if conditional?